lib/statsd/instrument.rb

```diff
@@ -523,7 +523,7 @@ def service_check(
     collect_metric(:_sc, name, status, sample_rate: sample_rate, tags: tags, prefix: prefix, metadata: metadata)
   end
 
-  private
+  protected
```

Comment:
Why was this change needed?  `protected` doesn't seem like it makes sense on a singleton class.

Reply:
This was left over from an unsuccessful debugging attempt. It is not needed.